src/handlers/serenity.rs

```diff
@@ -253,7 +249,15 @@ impl SerenityHandler {
                 })
                 .create_application_command(|command| {
                     command.name("skip").description("Skips the current track")
```

Comment:
If we have an optional argument for the track index, this description should probably read "Starts a vote to skip **to a specific track**" or something more broad.

Reply:
You can find that in the `voteskip` command!

Reply:
I'm a little confused with the reply. I'm saying this description is sort of misleading because you can also skip up to a certain track, not just the currently playing.

![image](https://user-images.githubusercontent.com/5618493/155146841-e1484ba9-a4e0-4095-96c6-33b65647778a.png)

`voteskip`'s description is correct IMO!

Reply:
> I'm a little confused with the reply. I'm saying this description is sort of misleading because you can also skip up to a certain track, not just the currently playing.

Sure, but there is no vote with the skip command 😅 Additionally, the base command does skip the currently playing track, so I think it makes sense.